deployment/enterprise_sso/enterprise_aws_sso_stack.py

```diff
@@ -194,6 +194,28 @@ def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
             ],
         )
 
+        ## Assignment management role for assignment execution function ##
+        assignment_management_policy = iam.PolicyDocument(
```

Comment:
can you please change the name of this as it clashes with the same var name in enterprise_aws_sso_management_stack.py?

Reply:
I renamed the variable